test/unit/foreman/renderer/snapshots/ProvisioningTemplate/iPXE/iPXE intermediate script.snap.txt

```diff
@@ -5,167 +5,137 @@
 isset ${net0/mac} || goto no_nic
 dhcp net0 || goto net1
 chain http://foreman.some.host.fqdn/unattended/iPXE?mac=${net0/mac} || goto net1
-
```

Comment:
I guess we want to keep the newline here, right?

Reply:
could you fix the relevant template in community templates please? This is auto-generated from it :)

Reply:
https://github.com/theforeman/community-templates/pull/582